public/components/layout/LeftSidePanel.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <aside class="left-side-panel-container">
+  <aside class="left-side-panel-container d-flex flex-column">
```

Comment:
The variable-facets component expects to be consumes by a flex container. Without it, scrolling breaks because it overflows the view port. That said, there's a lot I can cut out in favour of bootstrap here, so I will.